include/api.php

```diff
@@ -3952,7 +3953,7 @@ function api_friendica_direct_messages_search($type){
 	function api_friendica_profile_show($type){
 		$a = get_app();
 
-		if (api_user()===false) throw new ForbiddenException();
+		if (api_user()===false) throw new HTTP\ForbiddenException();
```

Comment:
Standards: Please add braces to this conditional statement and spaces around the `===` operator.